app/webpacker/components/Results/resultsFilter.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from 'react';
+import {
+  Button, ButtonGroup, Form, Segment,
+} from 'semantic-ui-react';
+import { EventSelector } from '../wca/EventSelector';
+import { RegionSelector } from '../CompetitionsOverview/CompetitionsFilters';
+import { countries } from '../../lib/wca-data.js.erb';
+import I18n from '../../lib/i18n';
+
+export default function ResultsFilter({ filterState, filterActions }) {
+  const {
+    event,
+    region,
+    rankingType,
+    gender,
+    show,
+  } = filterState;
+
+  const {
+    setEvent,
+    setRegion,
+    setRankingType,
+    setGender,
+    setShow,
+  } = filterActions;
+
+  const regionIso2 = useMemo(() => {
+    if (region === 'world') {
+      return 'all';
+    }
+    return countries.real.find((country) => country.id === region)?.iso2 || region;
+  }, [region]);
+  return (
+    <Segment raised>
+      <Form>
+        <Form.Field>
+          <EventSelector
+            title={I18n.t('results.selector_elements.events_selector.event')}
+            selectedEvents={[event]}
+            onEventSelection={({ eventId }) => setEvent(eventId)}
+            hideAllButton
+            hideClearButton
+          />
+        </Form.Field>
+        <Form.Field>
+          <RegionSelector
+            region={regionIso2}
+            dispatchFilter={({ region: r }) => {
+              if (r === 'all') {
+                setRegion('world');
+              } else {
+                setRegion(countries.byIso2[r]?.id ?? r);
+              }
+            }}
+          />
+        </Form.Field>
+        <Form.Group widths="equal">
+          <Form.Field>
+            <label>{I18n.t('results.selector_elements.type_selector.type')}</label>
+            <ButtonGroup primary compact widths={2}>
```

Comment:
The second button is only there for non-mbf events though?

Reply:
I think it looks nicer though with widths={2} even for mbf
![image](https://github.com/user-attachments/assets/44a1fb51-a81c-4c85-94f5-86c0de72c6e2)
vs
![image](https://github.com/user-attachments/assets/6afacaba-9537-4bf5-b782-7235eb90ee2d)